frontend/src/components/MapPanel.tsx

```diff
@@ -75,6 +77,7 @@ function MapPanel({
     const box = extractBoundingBox();
     if (box.length > 0) {
       onBoundingBoxChange(extractBoundingBox());
+      (searchBox as any).current = getSearchedAreaBox();
```

Comment:
Is the cast needed?

Reply:
yeah, it does the searchbox doesn't have 'current' thing that we ran into with the map before